src/hooks/useMultiSelect.stories.mdx

```diff
@@ -12,7 +12,7 @@ import { getCategory } from "../utils/componentCategories";
 | Latest       | `1.1.0`    |
 
 This custom hook, inspired by [downshift.js](https://www.downshift-js.com/),
-exposes functions used for a MultiSelect components. `useMultiSelect` provides all functions needed to handle the selectedItems, this hook can be used for
+exposes functions used for MultiSelect components. `useMultiSelect` provides all functions needed to handle the selectedItems, this hook can be used for
```

Comment:
To make this a bit clearer, change to the following…

```
…exposes functions used for `MultiSelect` components. `useMultiSelect` provides all functions needed to handle the `selectedItems` prop. This hook can be used for...
```

Reply:
changed